### DISCUSSION TITLE
传扬DM神教😎😎😎

### DISCUSSION THREAD

**Discussion by isle200inf** [share121/inter-knot#2773] [其他]
![图片](https://f4.bcbits.com/img/a2145936506_10.jpg)

# [Stained, Brutal Calamity](https://dlink.host/musics/aHR0cHM6Ly9vbmVkcnYtbXkuc2hhcmVwb2ludC5jb20vOnU6L2cvcGVyc29uYWwvc3Rvcl9vbmVkcnZfb25taWNyb3NvZnRfY29tL0VjVlZ5YlhnTjhCRnRzU2duQjQ3YkpnQkJjdkZnbnV2a25oU2lUWVotN0FVUGc.mp3)

## Lyrics
> -=Grief - Perspective of Calamitas=-
No more holding back
They'll wish for demise
They thought they were invincible
But you could see through their hollow lies
You've seen how far they've come
How many they've slain
You will gladly tear their limbs apart
All before they start to kill again
>  
> They were seen as a threat by the moment the dragon was killed by their hand
Now you're sent to demolish the foe upon your ruler's despondent demand
>  
> Something tells you they have scaled in their power
Should they win, it shall be your final hour
But you still have control of the fight
If they mess up, you might have a chance to destroy them
>  
> But is this the right path that you've taken?
Are your morals correct, or mistaken?
Are you sure you want to lose your mind
From a legend whose power could salvage mankind?
>  
> -=Lament - Perspective of the Player=-
You have known for certain
Your path is one of justice
All those who try to obstruct shall have paid the ultimate price
Yet, upon inspection
Your foe is overwhelming
But you remember what she tried to sacrifice
>  
> Signs of self-disappointment start infiltrating
(These putrid emotions begin to take control)
Symptoms of paranoia begin invading
(Such petty emotions have sabotaged your soul)
Your mental fortitude is fading
Losing trust, ending friendship, and rage escalating
>  
> Right on the road to your sadness
Watching closely as courage turns into madness
Are you sure you'd want to lose your life
From a witch who had caused you nothing but strife?
>  
> -=Epiphany=-
The end has come;
The witch and the prophecy
Deciding fates with the world on the line
Who shall emerge
From the ashes of calamity?
How shall this fable end through these legends intertwined?
>  
> End of reasoning
Carnage and pain
A feral sense of dominance
As all the blood melts into the rain
A clash of history
In blood, sweat and tears
This war will cause our world to change
And will be remembered through the years

**Comment by zhizhizhiwang** [comment]
抽象

**Comment by HSTTH** [comment]
嚯
